include/SimCore/KaonPhysics.h

```diff
@@ -62,26 +67,29 @@ class KaonPhysics : public G4VPhysicsConstructor {
    *
    * The processes are
    *
-   * K^0_L -> \pi^0 + \pi^0 + \pi^0
-   * K^0_L -> \pi^0 + \pi^+ + \pi^-
    * K^0_L -> \pi^- + e^+ + \nu_e
    * K^0_L -> \pi^+ + e^- + \nu_e
+   * K^0_L -> \pi^0 + \pi^0 + \pi^0
    * K^0_L -> \pi^- + \mu^+ + \nu_\mu
    * K^0_L -> \pi^+ + \mu^- + \nu_\mu
+   * K^0_L -> \pi^0 + \pi^+ + \pi^-
    *
    * and
    *
    * K^0_S -> \pi^+ + \pi^-
    * K^0_S -> \pi^0 + \pi^0
    *
+   * @note: The order in the the decay table is sorted by the branching ratios
```

Comment:
Perhaps this should be "must be" rather than "is", if I understood the bug correctly? If I'm right, I think the stronger language will be more helpful for the future developer reading the comment

Reply:
and same on L52